scss/crux.scss

```diff
@@ -50,3 +50,8 @@ mark {
 .html-comma:not(:first-child):before {
   content: ", ";
 }
+
+.font-resize>.open {
```

Comment:
what is this for ?

Reply:
Added Timezone Component in React-Crux. It has some errors while display the list. So added custom Style for that component